docs/false_positives.md

```diff
@@ -130,6 +131,22 @@ Other macros or builtins expressing unreachable code may be used.
 Note that the rewritten code is also safer, since debug builds now check for
 more precondition violations. 
 
+In case of C++11 or later, another option is to use Immediately-Invoked
+Function Expression (IIFE) to avoid assigning a meaningless value.
+
+```cpp
+int f(MyEnum Val) {
+  const int x = [&] {
```

Comment:
This code snipper looks too much alike the previous example. I think you should put a comment in this line (and at the call site of the lambda) to mark that this is the difference.